app/src/main/java/com/example/busschedule/ui/BusScheduleScreens.kt

```diff
@@ -162,19 +172,33 @@ fun BusScheduleScreen(
     } else {
         "$stopName ${stringResource(R.string.route_stop_name)}"
     }
-
-    Column(modifier) {
+    val layoutDirection = LocalLayoutDirection.current
+    Column(
+        modifier = modifier.padding(
+            start = contentPadding.calculateStartPadding(layoutDirection),
+            end = contentPadding.calculateEndPadding(layoutDirection),
+        )
+    ) {
         Row(
             modifier = Modifier
                 .fillMaxWidth()
-                .padding(dimensionResource(R.dimen.padding_medium)),
+                .padding(
+                    top = contentPadding.calculateTopPadding(),
```

Comment:
nit: Rearrange arguments so they match the function signature.

<img width="705" alt="Screenshot 2023-08-31 at 3 51 21 PM" src="https://github.com/google-developer-training/basic-android-kotlin-compose-training-bus-schedule-app/assets/7107817/9ce37c53-2cda-4ab4-b885-26bfecd354cb">